Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -12,18 +12,18 @@ import Mathlib.Analysis.SpecialFunctions.Pow.Deriv
 # Divide-and-conquer recurrences and the Akra-Bazzi theorem
 
 A divide-and-conquer recurrence is a function `T : ℕ → ℝ` that satisfies a recurrence relation of
-the form `T(n) = ∑_{i=0}^{k-1} a_i T(r_i(n)) + g(n)` for large enough `n`, where `r_i(n)` is some
-function where `‖r_i(n) - b_i n‖ ∈ o(n / (log n)^2)` for every `i`, the `a_i`'s are some positive
-coefficients, and the `b_i`'s are reals `∈ (0,1)`. (Note that this can be improved to
-`O(n / (log n)^(1+ε))`, this is left as future work.) These recurrences arise mainly in the
-analysis of divide-and-conquer algorithms such as mergesort or Strassen's algorithm for matrix
-multiplication.  This class of algorithms works by dividing an instance of the problem of size `n`,
-into `k` smaller instances, where the `i`-th instance is of size roughly `b_i n`, and calling itself
-recursively on those smaller instances. `T(n)` then represents the running time of the algorithm,
-and `g(n)` represents the running time required to actually divide up the instance and process the
-answers that come out of the recursive calls. Since virtually all such algorithms produce instances
-that are only approximately of size `b_i n` (they have to round up or down at the very least), we
-allow the instance sizes to be given by some function `r_i(n)` that approximates `b_i n`.
+the form `T(n) = ∑_{i=0}^{k-1} a_i T(r_i(n)) + g(n)` for sufficiently large `n`, where `r_i(n)` is
+a function where `‖r_i(n) - b_i n‖ ∈ o(n / (log n)^2)` for every `i`, the coefficients `a_i` are
```

Comment:
how about "a function such that ..."? (I personally prefer this over both the existing version and this rephrasing.)

Reply:
I like your proposal. It's now incorporated. Unfortunately it made the diff worse, to the point where you now no longer get the real changes highlighted. I wish github understood this as just a single block of text. Oh well.